src/social/components/Comment/CommentText.js

```diff
@@ -1,32 +1,38 @@
 import React, { useState, useMemo } from 'react';
 import { FormattedMessage } from 'react-intl';
 import Truncate from 'react-truncate-markup';
-import { findChunks } from '~/helpers/utils';
+import Markdown from 'markdown-to-jsx';
 import MentionHighlightTag from '~/core/components/MentionHighlightTag';
-import ChunkHighlighter from '~/core/components/ChunkHighlighter';
-import Linkify from '~/core/components/Linkify';
+import { formatMentionChunks } from '~/core/components/ChunkHighlighter';
 import { CommentContent, ReadMoreButton } from './styles';
 
 const COMMENT_MAX_LINES = 8;
 
 const CommentText = ({ text, className, mentionees, maxLines = COMMENT_MAX_LINES }) => {
```

Comment:
mentionees?

Reply:
It is an array of mentioned people. Not sure it is an actual English word though :D